### ISSUE TITLE
[Discussion] Wallet - User goes to Plaid after filling out the debit card fields

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#37265]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.44-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Email or phone of affected tester (no customers):** applausetester+022624dr003@applause.expensifail.com
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal Team

## Action Performed:
1. Open App or go to staging.new.expensify,com
2. Log in with expensifail account
3. Go to Wallet
4. Select "Add bank account" > "Debit card"
5. Fill in all fields using the card number from the Sandbox Test Cards list
6. Tap/click "Save"

## Expected Result:
Debit card saved

## Actual Result:
User goes to Plaid
(If bank accounts were previously added, then user will be asked to fill in additional details instead of Plaid).
But the card is added.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/a97bd5b2-5666-4cb5-a53d-8198390b95ea

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by izarutskaya** [comment]
@strepanier03 I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors.

**Comment by izarutskaya** [comment]
We think that this bug might be related to #wave8-collect-admins
CC @zanyrenney

**Comment by strepanier03** [comment]
Thanks @izarutskaya - I'm working on testing this and will have a little internal chat, then move this forward tomorrow.

**Comment by strepanier03** [comment]
Discussion ongoing [here](https://expensify.slack.com/archives/C03U7DCU4/p1709321449357769).

**Comment by strepanier03** [comment]
tl;dr: Adding a debit card is useless unless the wallet is enabled. The only way to enable the wallet is to add a bank account. This means that adding the debit card is prompting Plaid so that the wallet can be enabled and be useful. 

I'll update when I have a final decision on how to proceed.

**Comment by strepanier03** [comment]
Discussion is still ongoing.

**Comment by strepanier03** [comment]
Okay, we are going to close this for the following reason. 

Adding a debit card, it is unusable unless the user also enables the wallet. Enabling the wallet requires adding the bank account (Plaid step). When you add a debit card, we prompt you to add your bank account via Plaid so that it can be used. There is currently no way the user would know that they have to enable the wallet to use the card. 

Because of this, we're going to leave the prompts to add the bank account and enable the wallet after adding a debit card. This may change in the future if we create a way to alert customers to the need to enable the wallet. If we do that, we'd then allow adding the debit card without enabling the wallet via Plaid first.
